databases/week2/Week2HWTASK.sql

```diff
@@ -0,0 +1,56 @@
+-- Part 1: Working with tasks
+-- Add a task
+select * from task;
+INSERT INTO task 
+VALUES 
+(36,"Do MYSQL HYF exercises","Read the exercises and resolve it","2018-10-10 10:30:05","2018-10-09 10:30:05","2018-10-20 11:30:05",1);
+
+ -- Change the title of a task
+ALTER TABLE task RENAME COLUMN title TO task_title;
+UPDATE task set task_title = "Play music"  WHERE id=11;
+-- Change a task due date
+UPDATE task set due_date = "2017-11-20 23:58:03" WHERE id=2;
+-- Change a task status
+UPDATE task set status_id = 3 WHERE id=1;
+-- Mark a task as complete
+UPDATE task set status_id = 3 WHERE task_title="Charge electric bicycle" AND id=22;
+-- Delete a task
+DELETE FROM task where id=22;
+
+-- Part 2: School database
+-- Creating class table
+CREATE TABLE class(id INT PRIMARY KEY,name VARCHAR(50),begins DATE,ends DATE);
```

Comment:
A better way to write this query is:

```
CREATE TABLE class(
    id INT PRIMARY KEY,
    name VARCHAR(50),
    begins DATE,
    ends DATE);
```
Makes it easier to read.

Other than that:
 -  i'd recommend making the name longer, names can be really long, so i'd set the VARCHAR to 255 (the maximum length as far as i know),
 - i'd add NULL / NOT NULL to each field, to make it explicit whether or not the fields can be NULL
 - it is a good idea to set the `id` to an unsigned, since the `id` is always a positive number, if you set it to unsigned, you can have higher `id`values, see [here](https://dev.mysql.com/doc/refman/8.0/en/integer-types.html) 
 - you should always set the `id` to AUTO_INCREMENT, so that you never have to set it explicitly, because to define the `id` explicitly you first need to know what's the highest `id` value in the table, and it's not worth the trouble :)

Reply:
Thanks for the detailed explanation, changes made.